configuration/server.ini

```diff
@@ -63,15 +64,24 @@ description = Deny all administrator accounts.
 usernames = root, adm, admin, administrator
 
 
-[databases/default-sqlite-db-uuid]
+[authentications/ban-ip-uuid]
 enabled = Yes
+type = ip-time-ban
+name = Ban IP with multiple failures
+description = Will ban the source IP for a certain time on multiple failures.
+ban_interval = 600
+ban_after_count = 10
+
+
+[databases/default-sqlite-db-uuid]
```

Comment:
This section can be removed as [databases/] are no longer used in v4.

Or maybe replace it with `[resources/DEFAULT-SQLITE]`

I will commit an update to the config